models/src/node/mod.rs

```diff
@@ -3,7 +3,7 @@ mod error;
 
 pub use self::client::*;
 pub use self::error::Error as BottlerocketNodeError;
-use self::error::{Error, Result};
+use self::error::Error;
```

Comment:
Curious why do you remove the Result but switch to full path here?

Reply:
With the newest version of `kube-rs`, the `CustomResource` macro didn't seem to like it if we changed the meaning of `Result`.